t/23-baseclass.t

```diff
@@ -193,6 +194,15 @@ subtest 'SSH utilities' => sub {
 
         is($baseclass->check_ssh_serial(23), 0, 'Return 0 if SSH serial isn\'t connected');
     };
+
+    subtest Run => sub {
+        path('base_state.json')->remove;
```

Comment:
Does this when the file does not exist?

Reply:
Nothing happens if the file doesn't exist.